tests/playTests.sh

```diff
@@ -1549,6 +1565,7 @@ if [ -n "$hasMT" ]
 then
     println "\n===>   adaptive mode "
     roundTripTest -g270000000 " --adapt"
+    roundTripTest -g270000000 " --adapt=" && die "invalid compression parameter"
```

Comment:
This shouldn't be a `roundTripTest`.

Try moving this down after line 1573, and doing:

```
zstd --adapt= tmp && die "invalid compression parameter"
```

Reply:
It's like this.